modules/3d-tiles/src/lib/parsers/helpers/parse-3d-tile-subtree.ts

```diff
@@ -74,15 +94,17 @@ export default async function parse3DTilesSubtree(data: ArrayBuffer): Promise<Su
 async function getExplicitBitstream(
   subtree: Subtree,
   name: string,
-  internalBinaryBuffer: ArrayBuffer
+  internalBinaryBuffer: ArrayBuffer,
+  basePath: string
 ): Promise<ExplicitBitstream> {
   const bufferViewIndex = subtree[name].bufferView;
   const bufferView = subtree.bufferViews[bufferViewIndex];
   const buffer = subtree.buffers[bufferView.buffer];
 
   // External bitstream loading
   if (buffer.uri) {
-    const response = await fetchFile(buffer.uri);
+    const bufferUri = resolveUri(buffer.uri, basePath);
+    const response = await fetchFile(bufferUri);
```

Comment:
Should we be using `context.fetch` to make sure we don't get problems in workers etc? 

I am not 100% sure, but then we might not need to pass in basePath as I think it is already on context.

Reply:
Done